### PR TITLE
fix(web-vitals): Fix Discover drilldown when measurement values do not exist.

### PR DESCRIPTION
**Pull request by dashed**
Drilldown in a Discover query where measurement values are `null` will be appended to the search conditions. This should not occur.

Drilldown:

Screen Shot 2020-11-10 at 12 07 14 PM

After drilldown: 
Screen Shot 2020-11-10 at 12 07 25 PM

### CODE REVIEW

**Comment by Zylphrex** [comment]
Would it make more sense to generate a `has` condition in this situation instead?

**Comment by dashed** [comment]
@Zylphrex no because we still want to find events even if one of the measurement doesn't exist.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/views/eventsV2/utils.tsx [hunk 1]
```diff
@@ -294,6 +294,12 @@ function generateAdditionalConditions(
           ? String(value)
           : String(value).trim();
 
+      if (column.field.startsWith('measurements.') && !nextValue) {
```
Comment (Zylphrex) [inline src/sentry/static/sentry/app/views/eventsV2/utils.tsx:297]: I'd recomment using `isMeasurement` from `src/sentry/static/sentry/app/utils/discover/fields.tsx` to do this check.

Comment (dashed) [reply src/sentry/static/sentry/app/views/eventsV2/utils.tsx]: Should be addressed in https://github.com/getsentry/sentry/pull/21905/commits/0d76582eaaf379f279a795b5004591e90096da7c

**Pull request merged**